jqm-all/jqm-ws/src/main/react/src/components/Runs/RunsPage.tsx

```diff
@@ -468,11 +483,46 @@ const RunsPage: React.FC = () => {
                                     setShowDetailsJobInstanceId(
                                         jobInstanceId
                                     );
+                                    setStdType("NONE");
                                 }}
                                 size="large">
                                 <DescriptionIcon />
                             </IconButton>
                         </Tooltip>
+                        <Tooltip
+                            key={"Log stdout"}
+                            title={"Log stdout"}
+                        >
+                            <IconButton
+                                color="default"
+                                aria-label={"Log stdout"}
+                                onClick={() => {
+                                    setShowDetailsJobInstanceId(
+                                        jobInstanceId
+                                    );
+                                    setStdType("STDOUT");
+                                }}
+                                size="large">
```

Comment:
With the two added ones the action buttons take a lot of space now. 
Can you set the size to "small" for all of them?

Reply:
Could you also add a tooltip for the restart job button ?